test/parallel_api/numeric/numeric.ops/transform_reduce.pass.cpp

```diff
@@ -163,9 +163,11 @@ main()
                             [](const float32_t& a, const float32_t& b) -> float32_t { return a + b; },
                             [](const float32_t& x) -> float32_t { return x + 2; },
                             [](::std::size_t) -> float32_t { return rand() % 1000; });
+// MyClass must not be used with DPC++ policies since it is not device copyable
+#if !TEST_DPCPP_BACKEND_PRESENT
```

Comment:
I am not sure this is the right fix to do. Looking at `MyClass` definition I don't understand at first glance why it's **not** trivially copyable. Is there any good reason?

Reply:
It is not trivially copyable because of having user-defined copy constructor, but we can make it default and thus keep such test case. Good catch, thank you!